js/source/image_source.js

```diff
@@ -31,7 +31,7 @@ module.exports = ImageSource;
  * map.removeSource('some id');  // remove
  */
 function ImageSource(options) {
-    this.urls = options.urls;
+    this.url = options.url;
```

Comment:
⚠️  The documentation was wrong but this is still a breaking change. ⚠️ 

Can we avoid the headaches by writing `this.url = options.url || options.urls;`?